Lib/yawCalibration.h

```diff
@@ -15,7 +15,7 @@
 #ifndef YAWCALIBRATION_H_
 #define YAWCALIBRATION_H_
 
-#include "../Util/control_util.h"
+#include <stdbool.h>
```

Comment:
"..Util/control_util.h" should be included. The constants are now available by including "Putty.h"